### PR TITLE
Add common Makefile to share between recipes

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: recipes/common/Makefile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+CHROMADBIMAGE ?= quay.io/ai-lab/chromadb:latest
```
Comment (lmilbaum) [inline recipes/common/Makefile]: Which variable is used for the ghcr.io image name?

File: recipes/common/Makefile
Comment (lmilbaum) [inline recipes/common/Makefile:1]: I'd suggest naming it something like: Makefile.common. That will help developers to figure out its purpose.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: recipes/common/Makefile (hunk 1 above)
Comment (rhatdan) [reply recipes/common/Makefile]: I don't currently do anything with ghcri.io images.

Comment (lmilbaum) [reply recipes/common/Makefile]: The testing framework is consuming the images under tests from ghcr and expected to publish them to quay.io if tests are successful.

Comment (rhatdan) [reply recipes/common/Makefile]: Changing to 
.PHONY: run
run: 
	podman run -it -p 8501:8501 -e MODEL_SERVICE_ENDPOINT=http://10.88.0.1:8001/v1 ${APPIMAGE}

Comment (rhatdan) [reply recipes/common/Makefile]: The images should be built locally and pushed when tests pass, but not sure they should change names.

File: recipes/common/Makefile
Comment (rhatdan) [reply recipes/common/Makefile:1]: Good idea.

File: recipes/common/Makefile (hunk 1 above)
Comment (rhatdan) [reply recipes/common/Makefile]: None, I changed it to use ${APPIMAGE}

**Review by MichaelClifford — APPROVED** [review]
🙏 Thanks! I was starting to get worried by all the duplicate Makefiles 😄

**Pull request merged**